### DISCUSSION TITLE
Allow apache_port to be published on docker network

### DISCUSSION THREAD

**Discussion by mtthidoteu** [nextcloud/all-in-one#789] [Ideas] [answered]
### How to use GitHub
* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Is your feature request related to a problem? Please describe. 

If I wanted to use Docker Networks for my reverse proxy, this app does not let me do it. The reverse proxy must be on the Host's network which for most people is unsecure.

### Describe the solution you'd like

The apache_port variable should be on the host's network but also on the container's default network so that a reverse proxy can connect to Nextcloud_aio on it without going through the host

### Describe alternatives you've considered

na

### Additional context

Related to #787

**Comment by szaimen** [comment]
Moving to discussion since this is an non-existing feature.

**Comment by szaimen** [comment]
Not sure if this will be implemented. As a workaround have a look at https://github.com/nextcloud/all-in-one/tree/main/manual-install which allows to customize most things.

**Comment by szaimen** [comment, marked as answer]
I will not work on this. However ip-bindings will come with the next release. See https://github.com/nextcloud/all-in-one/issues/889
